src/main/java/de/zalando/aruha/nakadi/config/NakadiConfig.java

```diff
@@ -75,6 +77,9 @@ public EventStreamFactory eventStreamFactory() {
     }
 
     @Bean
+    public Enrichment enrichment() { return new Enrichment(new EnrichmentsRegistry()); }
```

Comment:
I know we don't have coding guidelines, but I believe our unofficial approach is to split this to three lines.